### PR TITLE
Style and language review for the Service Binding section

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

### CODE REVIEW

File: docs/src/main/asciidoc/deploying-to-kubernetes.adoc [hunk 1]
```diff
@@ -1400,9 +1401,9 @@ quarkus.kubernetes-service-binding.services.db-demo.name=my-db
 
 === Automatic service binding
 
-The `quarkus-kubernetes-service-binding` extension can automatically generate the `ServiceBinding` resource after detecting that an application requires access to external services provided by available bindable Operators.
+The 'quarkus-kubernetes-service-binding' extension can generate the 'ServiceBinding' resource automatically after detecting that an application requires access to the external services that are provided by available bindable Operators.
```
Comment (gastaldi) [inline docs/src/main/asciidoc/deploying-to-kubernetes.adoc:1404]: The ` is used to preview the text as code, better keep it in this case

File: docs/src/main/asciidoc/deploying-to-kubernetes.adoc [hunk 2]
```diff
@@ -1493,9 +1495,9 @@ For cases in which you need to modify the generated `ServiceBinding` to fix a na
 
 The `service key` is usually the name of the service, for example the name of the datasource, or the name of the `mongo` client. When this value is not available, the datasource type, such as `postgresql`, `mysql`, `mongo`, is used instead.
 
-To avoid naming conflicts between different types of services, the `service key` can  be prefixed with the specific type, such as `postgresql-__<person>__`.
+To avoid naming conflicts between different types of services, prefix the 'service key' with a specific datasource type, such as 'postgresql-__<person>__`.
```
Comment (gastaldi) [inline docs/src/main/asciidoc/deploying-to-kubernetes.adoc:1498]: Same here

**Review by gastaldi — CHANGES_REQUESTED** [review]
Looks good overall, added some comments that I'd like to see addressed before we merge this

File: docs/src/main/asciidoc/deploying-to-kubernetes.adoc (hunk 2 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc:1498]: Yeah, sorry, You are right. These googledocs.. :) Fixing

**Comment by gastaldi** [comment]
Can you please squash your commits before we merge it?

**Comment by MichalMaler** [comment]
@iocanel Hello Ioannis. Please, could you check the snips example in Semi-automatic and Automatic sections?
I knew that you were updating them, but I missed these updates when I was creating my draft from Downstresm docs. I put what was originally in Upstream main, so it should be OK. Just checking so that I could update my Downstream docs too if needed.
Thank you so much!

**Comment by MichalMaler** [comment]
Oh sorry, it looks like I screwed something during the rebase. Will close this and create a new clean PR. My apologies

**Comment by MichalMaler** [comment]
New MR with all suggestions in a single commit:
https://github.com/quarkusio/quarkus/pull/24007
@iocanel @gastaldi

**Pull request closed without merging**